Wikipedia/Code/BatchEditToolbarViewController.swift

```diff
@@ -32,6 +33,17 @@ public final class BatchEditToolbarViewController: UIViewController {
         let heightConstraint = stackView.heightAnchor.constraint(equalTo: view.heightAnchor)
         NSLayoutConstraint.activate([centerXConstraint, centerYConstraint, widthConstraint, heightConstraint])
         
+        let separator = UIView()
```

Comment:
One small tweak would be to move separator and stack view addition to their own respective funcs. I don't always adhere to method single responsibility principle but am trying to be better at decomposition by default.